docs/4.4/enterprise/ssh-rbac.md

```diff
@@ -271,25 +261,57 @@ spec:
 ```
 
 
-## RBAC for Sessions
+## Teleport Resources
+
+RBACs lets teams limit what resources are available to teleport users. This can be
+helpful if you don't want regular users editing SSO `auth_connector` or creating and
+editing new roles `role`.
```

Comment:
RBACs to RBAC

Can we also insert `for example` for the next sentence, so that we don't imply editing sso and roles is the only thing we can limit. 
```
This can be helpful if for example, you don't want regular users editing SSO `auth_connector` or creating and
editing new roles `role`.
```